source/help/getting-started/organizing-conversations.rst

```diff
@@ -119,13 +133,13 @@ In addition to this, System Admins can also unarchive channels `via the CLI <htt
 Converting Public channels to Private channels (and vice versa)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Select the channel name at the top of the center pane to access the drop-down menu, then select **Convert to Private Channel**. Team and System Admins can convert Public channels to Private channels. Please note that default channels such as ``Town Square`` and ``Off-Topic`` cannot be converted to Private channels.
+Select the Public channel name at the top of the center pane to access the drop-down menu, then select **Convert to Private Channel**. Team and System Admins can convert Public channels to Private channels. Please note that default channels such as ``Town Square`` and ``Off-Topic`` cannot be converted to Private channels.
 
-System Admins can also access this setting in **System Console > Channels > Edit (Channel Configuration)**. Due to security concerns of sharing Private channel history, only System Admins can convert Private channels to Public. 
+When a channel is converted, its history and membership are preserved. Membership in a Private channel is by invitation only. Publicly shared files remain accessible to anyone with the link. 
 
-When a channel is converted, its history and membership are preserved. Membership in a Private channel is by invitation only. Publicly shared files remain accessible to anyone with the link.
+.. note::
 
-Note that conversion of Private channels to Public channels can only be performed by a System Admin via the System Console or via `CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-channel-modify>`__.
+  Due to security concerns of sharing Private channel history, only System Admins can convert Private channels to Public channels. System Admins can access this setting in **System Console > Channels > Edit (Channel Configuration)**, or they can perform this action via `CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-channel-modify>`__.
```

Comment:
Should the CLI command be replaced with the mmctl command?

Reply:
Very good question, @justinegeffen! @esethna Can you confirm that we should update this documentation to reference an mmctl command instead?